docs/contributing/Localization In Compiler Tests.md

```diff
@@ -0,0 +1,42 @@
+Localization in Compiler Tests
+===
+
+The compiler tests are structured such that they can be run on a machine using any locale. This both serves as a tool to ensure the compiler does not have any localization errors but also to ensure that developers from any locale can contribute to the project. The ability to build and execute tests should not be limited to English speaking developers.
```

Comment:
>The compiler tests are structured such that they can be run on a machine using any locale. This both serves as a tool to ensure the compiler does not have any localization errors but also to ensure that developers from any locale can contribute  [](http://example.com/codeflow?start=0&length=245)

Consider breaking up long lines, not all tools do word-wrap  by default.

Reply:
Will clean up the word wrapping before merging.